pkgs/servers/mautrix-facebook/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  enableSystemd ? stdenv.isLinux,
+  fetchFromGitHub,
+  lib,
+  python3,
+  stdenv,
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "mautrix-facebook";
+  version = "v0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "tulir";
+    repo = pname;
+    rev = version;
+    sha256 = "0m7nznx3z6cg4wgvjybdivx22ifxcdri4i8501yibsri0jnpf0y2";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    CommonMark
+    aiohttp
+    asyncpg
+    mautrix
+    paho-mqtt
+    pillow
+    prometheus-client
+    pycryptodome
+    python-olm
+    python_magic
+    ruamel_yaml
+    unpaddedbase64
+    yarl
+  ] ++ lib.optional enableSystemd systemd;
+
+  doCheck = false;
+
+  prePatch = ''
+    sed -ie 's/^asyncpg.*/asyncpg>=0.20/' requirements.txt
+  '';
+
+  postInstall = ''
+    mkdir -p $out/bin
+
+    cat <<-END >$out/bin/mautrix-facebook
+    #!/bin/sh
+    PATH=${python3}/bin PYTHONPATH="$PYTHONPATH" exec python -m mautrix_facebook "\$@"
```

Comment:
Would it be better to `exec ${python3}/bin/python ...` rather than nuking PATH?

Reply:
I'm sure there's a different way of setting PYTHONPATH, but I cant remember it right now so /shrug

Reply:
Yeah, we should reference python exactly, I'll update that.. I copied this pythonpath setting from another mautrix package and it looks reasonable to me 🤷

Reply:
Yeah, looks good enough, and we can always improve later

Reply:
Please use makeWrapper.

Reply:
I don't think I can use makeWrapper in this case because the executable is python and imported as a module rather than an executable script.